MainModule/Server/Shared/Service.lua

```diff
@@ -937,6 +933,10 @@ return function(errorHandler, eventChecker, fenceSpecific, env)
 			})
 		end;
 
+		SanitizePattern = function(str)
+			return string.gsub(str, "([%^%$%(%)%%%.%[%]%*%+%-%?])", "%%%1")
```

Comment:
are you sure bro. this doesn't seem like good practise

Reply:
I'm not falling for another "trust me bro" smh!!!

Reply:
It litrally is the proper way to do it

It replaces all special characters with an encoded ver4sion

The square brackets denote a character class (the special characters) and the round brackets denote a string capture which is pasted to `%1`

`%n` denotes the n'th capture

Reply:
I like your funny words magic man